### PR TITLE
IfCurrent: moved from conditions to links

### PR DESCRIPTION
**Pull request by klimesf**
According to @dg['s suggestion](http://forum.nette.org/cs/17100-co-vylepsit-na-dokumentaci), macro {ifCurrent} needs moving from `conditions` to `links` section.

I'm not 100% sure this is right. I would like to hear your opinions on this, thanks in advance :)

EDIT: I will fix the EN version when we agree on a solution.

### CODE REVIEW

File: cs/default-macros.texy [hunk 1]
```diff
@@ -74,9 +73,10 @@ Zajímají vás bližší informace o [syntaxi Latte |templating#latte] nebo o [
...
 | Odkazy ||
 |---------------------------
-| `n:href`                       | "odkaz používaný v HTML elementech `<a>`":presenters#toc-odkazy-v-sablonach
-| `{link Presenter:action}`      | [vygeneruje odkaz |presenters#toc-odkazy-v-sablonach]
-| `{plink Presenter:action}`     | [vygeneruje odkaz na presenter |presenters#toc-odkazy-v-sablonach]
+| `n:href`                           | "odkaz používaný v HTML elementech `<a>`":presenters#toc-odkazy-v-sablonach
+| `{link Presenter:action}`          | [vygeneruje odkaz |presenters#toc-odkazy-v-sablonach]
+| `{plink Presenter:action}`         | [vygeneruje odkaz na presenter |presenters#toc-odkazy-v-sablonach]
+| `{ifCurrent $link} … {/ifCurrent}` | [speciální případ {if} pro aktivní odkaz |presenters#toc-odkazy-v-sablonach]
 |---------------------------
```
Comment (dg) [inline cs/default-macros.texy]: `speciální případ {if} pro aktivní odkaz` je naprosto nesrozumitelné… Nenapadá mě teď ideální formulace, ale asi cokoliv bude lepší ;)

Comment (klimesf) [inline cs/default-macros.texy]: Zatím jsem nepřepisoval text, řeším pouze umístění :) zkusím to trochu zlidštit

Comment (klimesf) [inline cs/default-macros.texy]: @dg Lepší? :)

**Comment by klimesf** [comment]
I have reworked the PR and added EN version, but I'm still not quite sure this is the best place to put `{ifCurret}`. Opinions welcome!

Ping @dg

**Comment by dg** [comment]
I like it

**Comment by klimesf** [comment]
If there are no other objections, I guess we can merge.

**Comment by dg** [comment]
Thank you!

**Pull request merged**